Resources/Locale/en-US/prototypes/catalog/fills/crates/engineering-crates.ftl

```diff
@@ -30,3 +30,6 @@ ent-CrateEngineeringMiniJetpack = Mini jetpack crate
 
 ent-CrateAirlockKit = Airlock kit
     .desc = A kit for building 6 airlocks, doesn't include tools.
+
+ent-CrateEvaKit = Eva kit
+    .desc = A set consisting of two prestigious EVA suits and helmets, tanks not included.
```

Comment:
```suggestion
    .desc = A set consisting of two prestigious EVA suits and helmets.
```

Reply:
for what?

Reply:
why would it be mentioned here?

Reply:
but the description literally says it contains only helmets and suits, is it for autistic players who cannot get it from reading for the first time?

Reply:
if you are gonna say theres no tanks include that there are no magboots or breath masks either, they are in the eva room too.